### DISCUSSION TITLE
What are the best Resize Filters?

### DISCUSSION THREAD

**Discussion by JonathonSunshine** [staxrip/staxrip#774] [Q&A]
Hey O!

Just curious which of the bundled staxrip resize filters are considered the best? (For making things smaller and bigger).

I normally use Spline64Resize because since it isn't alphabetical, I assumed that they are in order of quality.

But I'm also curious what the advanced ones are for

* Dither_Resize16
* Dither_Resize16 In Linear Light
* Hardware Encoder (I assume that is Nvidia Cuda etc)
* ResizeMT ( I presume this is just multithreaded versions of the other filters)
and finally
* SuperRes

**Comment by papa-pitufo** [comment]
Hi guy, it's a very good question, but I would expand the question

Donwnscale resize
Upscale resize

which are the best filter to each possibilities?

Waiting the news for the teachers.

**Comment by papa-pitufo** [comment]
In my test for upscaling (DVD to 720 or 1080) I use GaussResize.  Using X264

In my test for downscaling (4k to 2k) I use hardware encoding.  Using Nvenc
The same with upscaling (2k to 4k introducing HDR) I use hardware encoding also. Nvenc.
